build.rs

```diff
@@ -47,4 +47,10 @@ pub const RUN_TIME_CLOSURE: &str = "{runtime_closure}";
         "src/com.target.lorri.varlink",
         /* rustfmt */ true,
     );
+
+    // Generate src/com_target_lorri.rs
```

Comment:
```suggestion
    // Generate src/com_target_lorri_internal.rs
```

I think this calls for a `for_each` to process the two varlink files together, i.e. conceptually:

```
["src/com.target.lorri.varlink", "src/com.target.lorri.internal.varlink"]
    .for_each(|v| varlink_generator::cargo_build_tosource(v, /* rustfmt */ true);
```